src/footprints/rotary.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+    nets: ['A', 'B', 'C', 'S1', 'S2', 'MP1', 'MP2'],
+    params: {
+        class: 'S'
+    },
+    body: p => `
+        (module rotary_encoder (layer F.Cu) (tedit 603326DE)
+
+            ${p.at /* parametric position */}
+        
+            ${'' /* footprint reference */}
+            (fp_text reference "${p.ref}" (at 0 0.5) (layer F.SilkS) 
+                ${p.ref_hide} (effects (font (size 1 1) (thickness 0.15))))
+            (fp_text value rotary_encoder (at 0 8.89) (layer F.Fab)
```

Comment:
Value is usually left empty for a cleaner PCB.

Reply:
Removed in next commit